doc/source/gettingstarted.rst

```diff
@@ -0,0 +1,56 @@
+================
+Getting Started
+================
+
+Once you have freud `installed <installation.rst>`_, you can start using **freud** with any simulation data that you have on hand.
+As an example, we'll assume that you have run a simulation using the `HOOMD-blue <http://glotzerlab.engin.umich.edu/hoomd-blue/>`_ and used the ``hoomd.dump.gsd`` command to output the trajectory into a file ``trajectory.gsd``
+The GSD file format provides its own convenient Python file reader that offers access to data in the form of NumPy arrays, making it immediately suitable for calculation with **freud**.
+
+We start by reading the data into a NumPy array:
+
+.. code-block:: python
+
+    import gsd.hoomd, gsd.pygsd
+
+    f = gsd.pygsd.GSDFile(open('trajectory.gsd', 'rb'))
+    traj = gsd.hoomd.HOOMDTrajectory(f)
+
+
+We can now immediately calculate important quantities.
+Here, we will compute the radial distribution function :math:`g(r)` using the `freud.density.RDF` compute class.
+Since the radial distribution function is in practice computed as a histogram, we must specify the histogram bin widths and the largest interparticle distance to include in our calculation.
+To do so, we simply instantiate the class with the appropriate parameters and then pass in the data, taking advantage of **freud**'s *method chaining* API to do both at once:
```

Comment:
Let's introduce the non-chained syntax first. Chained syntax lines are too long and confusing to be the "hello world" snippet that we show new users.